File: translations/codenet/mistral_commenetd/C2Go/codellama/Code/s470461127.go
package main

import "fmt"

func main() {
	// Declare two integer variables x and y
	var x, y int
	x = 0
	y = 0

	// Read two integers from the user using fmt.Scanf function and store them in variables x and y respectively
	fmt.Scanf("%d", &x)
	fmt.Scanf("%d", &y)

	// Print the product of x and y and the sum of 2 times x and 2 times y using fmt.Printf function
	fmt.Printf("%d %d\n", x*y, (2*x) + (2*y))
}

File: translations/codenet/mistral_commenetd/C2P/starcoder/Code/s470461127.py
#include <stdio.h>

def main():
    # Declare two integer variables x and y
    x = 0
    y = 0

    # Read two integers from the user using scanf function and store them in variables x and y respectively
    x = int(input())
    y = int(input())

    # Print the product of x and y and the sum of 2 times x and 2 times y using printf function
    print(x * y, (2*x) + (2*y))

if __name__ == "__main__":
    main()

#